<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/home.png</normaloff>:/home.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QPushButton" name="btnCocina">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/sensor0.png) no-repeat center;
border: 0;</string>
   </property>
   <property name="text">
    <string>


Cocina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/sensor0.png) no-repeat center;
border: 0;
padding: 5px</string>
   </property>
   <property name="text">
    <string>


Habitación</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLiving">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/sensor0.png) no-repeat center;
border: 0;</string>
   </property>
   <property name="text">
    <string>


Living</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBanio">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/sensor0.png) no-repeat center;
border: 0;</string>
   </property>
   <property name="text">
    <string>


Baño</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
